### ISSUE TITLE
x/perf/internal/stats: test failure on s390x amd Golang 1.11 beta 1

### ISSUE THREAD

**Issue by eclipseo** [golang/go#26477]
This test failure only affects s390x architecture with Golang 1.11 beta 1. It was working before.
This is using the latest git.

```
Testing: "/builddir/build/BUILD/perf-2ce0818a26175593d58f0f5ddb67631b9918296b/_build/src/golang.org/x/perf/internal/stats"
+ GOPATH=/builddir/build/BUILD/perf-2ce0818a26175593d58f0f5ddb67631b9918296b/_build:/usr/share/gocode
+ go test -buildmode pie -compiler gc -ldflags '-extldflags '\''-Wl,-z,relro   -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '\'''
--- FAIL: TestNormalDist (0.00s)
    util_test.go:101: want {Mu:0 Sigma:1}.CDF(10000)=1, got 3.0600400278046567e+65
FAIL
exit status 1
FAIL	golang.org/x/perf/internal/stats	0.062s
```

**Comment by bcmills** [comment]
CC @mundaym

**Comment by mundaym** [comment]
This looks like a problem with the `Erfc` s390x assembly. The test doesn't pass with Go 1.9.x and Go 1.10.x either, so it's not a regression.

@billotosyr can you take a look and see if there is an easy fix for the assembly? Otherwise we should probably remove it for this release.

**Comment by randall77** [comment]
Yes, we should fix this one way or the other before the release.
We should also pull this test into the stdlib if we can.
I'm going to mark as a release blocker.

**Comment by billotosyr** [comment]
Working on this today. In this failing test, erfc is being called with an argument of ~ -7000, which is way way outside it's expected range of application. The go version of erfc passes because it tests if the input argument is out of reasonable range (-28 to 28). So to fix erfc I will also have to put such a test in. Not too hard to do, of course.

**Comment by randall77** [comment]
Can we get this fix in ASAP?  We'd like to do a release candidate on Wednesday.
